packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -80,6 +81,11 @@ class ExpansionTile extends StatefulWidget {
   /// Specifies if the list tile is initially expanded (true) or collapsed (false, the default).
   final bool initiallyExpanded;
 
+  /// Specifies padding for [title].
+  ///
+  /// When the value is null, padding is `EdgeInsets.symmetric(horizontal: 16.0)`.
```

Comment:
We can be more descriptive here as well to avoid some confusion. Something like: 
```
  /// Specifies padding for the [ListTile].
  ///
  /// Analogous to [ListTile.contentPadding], this property defines the insets for the
  /// [leading], [title], [subtitle] and [trailing] widgets. It does not inset the expanded
  /// [children] widgets.
  ///
  /// When the value is null, the tile's padding is `EdgeInsets.symmetric(horizontal: 16.0)`.
```